[BUG] Fails to build with Qt 6.10

**Describe the bug**
chiaki-ng fails to build when using Qt 6.10

**Debug Log**
see https://github.com/NixOS/nixpkgs/issues/454777
```
error: Cannot build '/nix/store/919kcxskd351p0lpmjpwjfild525jnf9-chiaki-ng-1.9.9.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/7vfa1a66iv7p679dfzz4sfd9px78ncl3-chiaki-ng-1.9.9
       Last 25 log lines:
       > -- Setsu disabled
       > -- Configuring done (8.4s)
       > CMake Error at gui/CMakeLists.txt:103 (target_link_libraries):
       >   Target "chiaki" links to:
       >
       >     Qt::GuiPrivate
       >
       >   but the target was not found.  Possible reasons include:
       >
       >     * There is a typo in the target name.
       >     * A find_package call is missing for an IMPORTED target.
       >     * An ALIAS target is missing.
       >
       >
       >
       > -- Generating done (0.3s)
       > CMake Warning:
       >   Manually-specified variables were not used by the project:
       >
       >     BUILD_TESTING
       >     CMAKE_EXPORT_NO_PACKAGE_REGISTRY
       >     CMAKE_POLICY_DEFAULT_CMP0025
       >
       >
       > CMake Generate step failed.  Build files cannot be regenerated correctly.
       For full logs, run:
         nix log /nix/store/919kcxskd351p0lpmjpwjfild525jnf9-chiaki-ng-1.9.9.drv
```

**To Reproduce**
Attempt to build with Qt 6.10

**Expected behavior**
Build completes successfully

**Desktop (please complete the following information):**
 - OS: NixOS
 - Version 25.11
 - Device Framework 13
 - chiaki-ng Version 1.9.9

**Additional context**
Seems to be due to a change in how `GuiPrivate` is found; build completes and runs with
```
diff --git a/gui/CMakeLists.txt b/gui/CMakeLists.txt
index 201dfaf4..b397ac19 100755
--- a/gui/CMakeLists.txt
+++ b/gui/CMakeLists.txt
@@ -7,7 +7,7 @@ if(APPLE)
        enable_language(OBJCXX)
 endif()

-find_package(Qt6 REQUIRED COMPONENTS Core Gui Concurrent Svg Qml Quick Widgets)
+find_package(Qt6 REQUIRED COMPONENTS Core Gui GuiPrivate Concurrent Svg Qml Quick Widgets)
 find_package(Qt6 COMPONENTS WebEngineQuick)
 find_package(Qt6 COMPONENTS DBus)
```

--- a/gui/CMakeLists.txt
+++ b/gui/CMakeLists.txt
@@ -8,6 +8,10 @@ if(APPLE)
 endif()
 
 find_package(Qt6 REQUIRED COMPONENTS Core Gui Concurrent Svg Qml Quick Widgets)
+if(Qt6_VERSION VERSION_GREATER_EQUAL "6.10.0")
+	find_package(Qt6 REQUIRED COMPONENTS GuiPrivate)
+endif()
+
 find_package(Qt6 COMPONENTS WebEngineQuick)
 find_package(Qt6 COMPONENTS DBus)
 

